src/components/Axis/Axis.tsx

```diff
@@ -44,7 +44,7 @@ export interface AxisProps {
   hiddenTicks?: boolean;
   tickPosition?: 'inner' | 'outer' | 'center';
   // TODO: Precise this.
-  tickLabelFormat?: () => string;
+  tickLabelFormat?: (any) => string;
```

Comment:
```suggestion
  tickLabelFormat?: TickLabelFormat;
```

Same in other places, and add:

```ts
export type TickLabelFormat = (value: number) => string;
```

in `src/types.ts`